fixtures/src/components/Header.js

```diff
@@ -0,0 +1,68 @@
+import { parse, stringify } from 'query-string';
+const React = window.React;
+
+const Header = React.createClass({
+  getInitialState() {
+    const query = parse(window.location.search);
+    const version = query.version || 'local';
+    const versions = [version];
+    return { version, versions };
+  },
+  componentWillMount() {
+    fetch('http://api.github.com/repos/facebook/react/tags')
+      .then(res => res.json())
+      .then(tags => {
+        let versions = tags.map(tag => tag.name.slice(1));
+        versions = ['local', ...versions];
+        this.setState({ versions });
+      });
+  },
+  handleVersionChange(event) {
+    const query = parse(window.location.search);
+    query.version = event.target.value;
+    if (query.version === 'local') {
+      delete query.version;
+    }
+    window.location.search = stringify(query);
+  },
+  handleFixtureChange(event) {
+    window.location.pathname = event.target.value;
+  },
+  render() {
+    return (
+    <header className="header">
+      <div className="header__inner">
+        <span className="header__logo">
+          <img src="https://facebook.github.io/react/img/logo.svg" alt="" width="32" height="32" />
+          React Sandbox (v{React.version})
+        </span>
+
+        <div className="header-controls">
+          <label htmlFor="example">
+            <span className="sr-only">Select an example</span>
+            <select value={window.location.pathname} onChange={this.handleFixtureChange}>
```

Comment:
Yeah, probably. The way I do it with version is to just pass it from the `window.location` and update `window.location.search` when it changes, forcing it to reload (since we need a full reload on version change).

I did the same thing here, but I guess we don't need a full reload on fixture change?